accounts/addproofer.php

```diff
@@ -175,8 +175,16 @@
 echo "<tr>";
 echo "  <th>" . _("E-mail Updates") . ":</th>";
 echo "  <td>";
-echo "    <input type='radio' name='email_updates' value='1' "; if($email_updates) echo "checked"; echo ">" . _("Yes") . "&nbsp;&nbsp;";
-echo "    <input type='radio' name='email_updates' value='0' "; if(!$email_updates) echo "checked"; echo ">" . _("No");
+echo "    <input type='radio' name='email_updates' value='1' ";
+if ($email_updates) {
+    echo "checked";
+}
+echo "    >" . _("Yes") . "&nbsp;&nbsp;";
+echo "    <input type='radio' name='email_updates' value='0' ";
+if (!$email_updates) {
+    echo "checked";
+}
+echo "    >" . _("No");
```

Comment:
The code obviously works the way it is, so this is just curiosity -- when should a label (like the "Yes" and "No") actually be wrapped in a `<label>` element?

Reply:
`<label>` elements are optional. They allow users to click on the text to access the associated checkbox/radio buttons in addition to the button themselves, but they are not required. We don't use them consistently in the code (indeed, we use them in relatively few places).

Reply:
Got it! Thanks for the clarification!

Reply:
I can take a look at enhancing these checkboxes once the formatting MR is resolved.